### PR TITLE
Bump MDN/Data NPM library

### LINKED ISSUE #94: Need to upgrade mdn-data to 2.0.1
[MDN/Data](https://github.com/mdn/data) updated their license in 2.0 to use a better open source license that doesn't break licensing issues for enterprise companies.

Unfortunately this change breaks the build and I'm not too familiar with the project to really dive in:

```bash
Write data to /Code/csstree/dist/default-syntax.json

/Code/csstree/lib/lexer/grammar/tokenizer.js:51
        throw new SyntaxParseError(message, this.str, this.pos);
        ^
SyntaxParseError: Expect `&`
    at Tokenizer.error (/Code/csstree/lib/lexer/grammar/tokenizer.js:51:15)
    at Tokenizer.eat (/Code/csstree/lib/lexer/grammar/tokenizer.js:42:18)
    at peek (/Code/csstree/lib/lexer/grammar/parse.js:415:23)
    at readImplicitGroup (/Code/csstree/lib/lexer/grammar/parse.js:327:20)
    at parse (/Code/csstree/lib/lexer/grammar/parse.js:483:18)
    at Object.get [as syntax] (/Code/csstree/lib/lexer/Lexer.js:157:36)
    at dumpMapSyntax (/Code/csstree/lib/lexer/Lexer.js:21:23)
    at Lexer.dump (/Code/csstree/lib/lexer/Lexer.js:317:25)
npm ERR! code ELIFECYCLE
```

### PR DESCRIPTION
**Pull request by acconrad**
Fixes #94

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "css-tree",
-  "version": "1.0.0-alpha.29",
```
Comment (lahmatiy) [inline package.json]: Please, don't change version

File: data/patch.json [hunk 2]
```diff
@@ -298,6 +298,18 @@
       "comment": "extend by non-standard width keywords https://developer.mozilla.org/en-US/docs/Web/CSS/width",
       "syntax": "<length> | <percentage> | auto | max-content | min-content | fit-content | fill-available | <-non-standard-width>"
     },
+    "motion": { 
+      "syntax": null
```
Comment (lahmatiy) [inline data/patch.json]: add `"comment": "remove when mdn/data released a version after 2.0.1"` for all added properties

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,9 @@
+## 1.1.0 (March 18, 2019)
```
Comment (lahmatiy) [inline CHANGELOG.md]: Please, don't set a version/date -> `## next`

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,9 @@
...
+
+- Upgrade `mdn-data` to use a better open-source license
```
Comment (lahmatiy) [inline CHANGELOG.md]: This is enough:
```
- Bumped `mdn/data` to `~2.0.1`
```

**Comment by acconrad** [comment]
@lahmatiy changes implemented - thanks for looking!

**Comment by acconrad** [comment]
all set @lahmatiy

**Comment by lahmatiy** [comment]
Thank you!

**Pull request merged**